mailpoet/lib/Config/Hooks.php

```diff
@@ -379,6 +380,16 @@ public function setupWooCommerceSubscriberEngagement() {
     );
   }
 
+  public function setupWooCommerceTracking() {
+    if ($this->wp->getOption('woocommerce_allow_tracking', 'no') === 'yes') {
+      $this->wp->addFilter(
+        'woocommerce_tracker_data',
```

Comment:
Just out of curiosity, as I'm unfamiliar with the tracker — do you know if this is executed by action scheduler or visitor requests?

Reply:
If I'm not mistaken, it uses WP Cron `wp_schedule_single_event`